drivers/bluetooth/hci/h5.c

```diff
@@ -397,6 +397,11 @@ static void h5_process_complete_packet(u8_t *hdr)
 		net_buf_put(&h5.rx_queue, buf);
 		break;
 	case HCI_EVENT_PKT:
+		if (buf->len && bt_hci_evt_is_prio(*((u8_t *)buf->data))) {
```

Comment:
`net_buf.data` is already defined as `u8_t *`, so this cast should not be necessary.

p.s. it would be cleaner to use `struct bt_hci_evt_hdr` to access the `evt` value.